microsetta_interface/implementation.py

```diff
@@ -923,7 +923,12 @@ def get_authrocket_callback(token=None, redirect_uri=None):
             "account_id": primary['account_id'],
             "email": primary['email']
         }
-        session[LANG_KEY] = primary["language"]
+        # NB: Disabling Japanese for the initial relaunch period until the
+        # Tokyo Tech team can review finalized translations
+        if primary["language"] == "ja_JP":
```

Comment:
Is there risk of exposure to other variations of the localization tag (e.g. just "ja")?

Reply:
No, `primary["language"]` is coming out of the `ag.account` table which will always be the standardized locale value.